src/nvim/testdir/Makefile

```diff
@@ -34,14 +34,12 @@ NEW_TESTS_ALOT := test_alot_utf8 test_alot
 NEW_TESTS_IN_ALOT := $(shell sed '/^source/ s/^source //;s/\.vim$$//' test_alot*.vim)
 # Ignored tests.
 # test_alot_latin: Nvim does not allow setting encoding.
-# test_arglist: ported to Lua, but kept for easier merging.
 # test_autochdir: ported to Lua, but kept for easier merging.
 # test_eval_func: used as include in old-style test (test_eval.in).
 # test_listlbr: Nvim does not allow setting encoding.
 # test_largefile: uses too much resources to run on CI.
 NEW_TESTS_IGNORE := $(NEW_TESTS_IN_ALOT) $(NEW_TESTS_ALOT) \
   test_alot_latin \
-  test_arglist \
```

Comment:
```
From test_arglist.vim:
Found errors in Test_argv():
Caught exception in Test_argv(): Vim(call):E118: Too many arguments for function: argc @ function RunTheTest[35]..Test_argv, line 20
```

Reply:
Introduced in https://github.com/neovim/neovim/commit/052ced4954075eca360ff7689afea82252f1c599